### DISCUSSION TITLE
Rust / Cargo is completely broken

### DISCUSSION THREAD

**Discussion by hamiltop** [aquasecurity/trivy#5308] [Bugs]
### Description

Raised in https://github.com/aquasecurity/trivy/issues/5214 and https://github.com/aquasecurity/trivy-db/issues/353

Something is crossed up in categorizing GHSA as `rust` vs `cargo` vulnerabilities. Because they are mistagged, they are ignored and no rust vulnerabilities are reported.

### Desired Behavior

When I run a scan in a rust project with a known vulnerability, I expect it to be flagged.

### Actual Behavior

Nothing is raised in rust projects.

### Reproduction Steps

```bash
See https://github.com/aquasecurity/trivy/issues/5214
```

### Target

Git Repository

### Scanner

Vulnerability

### Output Format

JSON

### Mode

Standalone

### Debug Output

```bash
See https://github.com/aquasecurity/trivy/issues/5214
```

### Operating System

Any

### Version

```bash
See https://github.com/aquasecurity/trivy/issues/5214
```

### Checklist

- [ ] Run `trivy image --reset`
- [ ] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by knqyf263** [comment]
Fixed in https://github.com/aquasecurity/trivy-db/pull/355
